thrimbletrimmer/scripts/common.js

```diff
@@ -579,12 +599,16 @@ function renderSystemMessages(chatMessageData) {
 
 	const sendTimeElement = document.createElement("div");
 	sendTimeElement.classList.add("chat-replay-message-time");
-	sendTimeElement.innerText = videoHumanTimeFromDateTime(chatMessageData.when);
+	sendTimeElement.innerText = chatMessageData.displayWhen;
 
 	const systemTextElement = document.createElement("div");
 	systemTextElement.classList.add("chat-replay-message-text");
 	systemTextElement.classList.add("chat-replay-message-system");
-	systemTextElement.appendChild(document.createTextNode(chatMessage.tags["system-msg"]));
+	let systemMsg = chatMessage.tags["system-msg"];
+	if (!systemMsg && chatMessage.tags["msg-id"] === "announcement") {
+		systemMsg = "Announcement";
```

Comment:
```suggestion
		systemMsg = `Announcement: ${chatMessage.params[1]}`;
```
This also shows the annoucement text.

Reply:
Will add it myself post-merge.

Reply:
oh, i see, you're adding it as a second message below. carry on.

Reply:
yeah, that's already how the sub messages show, so I just kept it the same thing